SonarQube.TeamBuild.PreProcessor/Interfaces/IPropertiesFetcher.cs

```diff
@@ -1,22 +0,0 @@
-﻿//-----------------------------------------------------------------------
-// <copyright file="IPropertiesFetcher.cs" company="SonarSource SA and Microsoft Corporation">
-//   Copyright (c) SonarSource SA and Microsoft Corporation.  All rights reserved.
-//   Licensed under the MIT License. See License.txt in the project root for license information.
-// </copyright>
-//-----------------------------------------------------------------------
-
-using SonarQube.Common;
-using System.Collections.Generic;
-
-namespace SonarQube.TeamBuild.PreProcessor
-{
-    public interface IPropertiesFetcher
```

Comment:
No longer required. The implementation of this class did very little; it's being replaced by a method on ISonarQubeServer.